new-frontiers-14/frontier-station-14
# New Ship: NT Meteor

## About the PR
This PR adds a new small expeditionary ship in the salvage category named the NT Meteor. It’s packed with salvage equipment, cargo space, and tightly built rooms, enabling it to function very well as a ship for players who like to salvage, mine, and go on expeditions.

## Why / Balance
This ship provides a solid alternative to the Pathfinder, which currently is the only small expeditionary ship in the game. I primarily found myself frustrated with the extra work required to operate/improve the Pathfinder, as well as its unwieldy size, poor layout, and single blast door setup. The Meteor, in contrast, sets itself up to be more sectioned off per department than the Pathfinder, with fewer total rooms, but each room fulfilling its purpose much more single-mindedly. In total, it provides an alternative for small crews that desire a more segmented setup of departments and efficient layout.

Otherwise, the NT Meteor takes up a 17x14 space and consumes around 16,925 Watts at rest. As well, it has a purchasing price of 51,525 spesos due to its small size, and a base sell price of 28,478 spesos.

## How to test
Start a local instance, warp to/spawn at the Expeditionary Lodge, purchase the NT Meteor at the expeditionary shipyard console, and see what you think of it.

## Media
These images display how the ship will be seen by players, how it appears with fullbright enabled, and with both fullbright and viewsubfloor enabled respectively.

Normal Lighting
![meteor final](https://github.com/user-attachments/assets/b061ec31-85af-41bb-9aea-f1e57f7d6837)

Fullbright
![meteor final fullbright](https://github.com/user-attachments/assets/93812195-2415-462a-bea0-4c7f8a8393e2)

Fullbright and Viewsubfloor
![meteor final fullbright subfloor](https://github.com/user-attachments/assets/0800f45a-46cb-49c6-95f9-c185be0a7085)

**Changelog**
The048
-add: Added the NT Meteor
